datasets/ncbi-sra.yaml

```diff
@@ -1,8 +1,8 @@
 Name: NIH NCBI Sequence Research Archive (SRA) on AWS
-Description:  "The Sequence Read Archive (SRA), produced by the [National Center for Biotechnology Information (NCBI)]](https://www.ncbi.nlm.nih.gov/) at the [National Library of Medicine (NLM)](http://nlm.nih.gov/) at the [National Institutes of Health (NIH)](http://nlm.nih.gov/), stores raw DNA sequencing data and alignment information from high-throughput sequencing platforms. The SRA provides open access to these biological sequence data to support the research community's efforts to enhance reproducibility and make new discoveries by comparing data sets. This bucket contains public SRA data in  original format from select high value and newly-released studies."
-Contact: "[SRA Staff](sra@ncbi.nlm.nih.gov)"
+Description:  "The Sequence Read Archive (SRA), produced by the [National Center for Biotechnology Information (NCBI)](https://www.ncbi.nlm.nih.gov/) at the [National Library of Medicine (NLM)](http://nlm.nih.gov/) at the [National Institutes of Health (NIH)](http://nlm.nih.gov/), stores raw DNA sequencing data and alignment information from high-throughput sequencing platforms. The SRA provides open access to these biological sequence data to support the research community's efforts to enhance reproducibility and make new discoveries by comparing data sets. This bucket contains public SRA data in  original format from select high value and newly-released studies."
+Contact: "[sra@ncbi.nlm.nih.gov](sra@ncbi.nlm.nih.gov)"
```

Comment:
You can make this just the email, no need for markdown. See https://github.com/awslabs/open-data-registry/blob/master/datasets/noaa-nexrad.yaml#L4 as an example

Reply:
Thanks for the tip. I made the change you suggested.